machine/quanta/quanta_common_rglbmc/busybox/patches/onie-sys-eeprom-platform.patch

```diff
@@ -0,0 +1,609 @@
+Add support for onie-syseeprom and mb_detect
```

Comment:
Appreciate for the suggestion. I will reorganize this patch. There are still other common platforms use the same mechanism to detect the corresponding platform.